src/components/Menu/MenuBar.vue

```diff
@@ -142,6 +152,9 @@ export default {
 				children: this.hiddenEntries,
 			}
 		},
+		fileName() {
+			return this.relativePath.substring(this.relativePath.lastIndexOf('/') + 1)
+		},
```

Comment:
You do not need the prop `repativePath`

https://github.com/nextcloud/text/blob/4c7baddf9631a80f27a34ca189fcc1c36306902e/src/components/Editor.provider.js#L68-L79

Reply:
It's default value. Can you please elaborate?